src/ranking_watcher/usma.rs

```diff
@@ -0,0 +1,152 @@
+use std::convert::TryFrom;
+use std::error::Error;
+use std::result;
+
+use reqwest;
+use scraper::{node::Element, node::Node, ElementRef, Html, Selector};
+use selectors::attr::CaseSensitivity;
+
+const RANKING_URL: &str = "https://ranking.cvgo.re/";
+
+type Result<T> = result::Result<T, Box<dyn Error>>;
+
+#[derive(Debug, Eq, PartialEq)]
+pub struct RankingEntry {
```

Comment:
Would it possibly make sense to make this more generic? I.e. move that to a separate file, so that watchers for other rankings (MERS, for example?) could be easily added.